Library/Formula/devil.rb

```diff
@@ -1,11 +1,53 @@
 class Devil < Formula
   desc "Cross-platform image library"
   homepage "https://sourceforge.net/projects/openil/"
-  url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/DevIL-1.7.8.tar.gz"
-  sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
   revision 1
 
-  bottle :disable, "Can't generate bottles until builds with either Clang or GCC-5"
+  stable do
+    url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/" \
+      "DevIL-1.7.8.tar.gz"
+
+    sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
+
+    # fix compilation issue for iluc.c
+    patch \
+      "--- a/src-ILU/ilur/ilur.c 2009-03-08 08:10:12.000000000 +0100\n" \
+      "+++ b/src-ILU/ilur/ilur.c 2010-09-26 20:01:45.000000000 +0200\n" \
+      "@@ -1,6 +1,7 @@\n" \
+      " #include <string.h>\n" \
+      " #include <stdio.h>\n" \
+      "-#include <malloc.h>\n" \
+      "+#include <stdlib.h>\n" \
+      "+#include \"sys/malloc.h\"\n" \
+      "\n" \
+      " #include <IL/il.h>\n" \
+      " #include <IL/ilu.h>\n"
+  end
+
+  head do
+    url "https://github.com/DentonW/DevIL.git"
+
+    depends_on "automake" => :run
```

Comment:
So in the HEAD version I had to add the call to "DevIL/autogen.sh" which initially failed because of missing auto tools on my machine. When I added the automake dependency this worked but then the audit run indicated that I should add "=> :build" to the automake dependency. When I did the latter the call to autogen.sh failed as if the auto tools where not in the path. At this point I changed "=> :build" to "=> :run" and everything worked again.

I suspect what I need is something like "=> :install" such thing is not available and I don't have enough domain knowledge yet to introduce it.